x/avs/keeper/keeper.go

```diff
@@ -21,26 +23,29 @@ type (
 	Keeper struct {
 		cdc            codec.BinaryCodec
 		storeKey       storetypes.StoreKey
-		operatorKeeper delegationtypes.OperatorKeeper
+		operatorKeeper operatortypes.Keeper
```

Comment:
Better to make this an interface in `expected_keepers.go`

Reply:
fixed